samples/__tests__/e2e.test.ts

```diff
@@ -25,150 +25,153 @@ const bobConfig: InitConfig = {
   autoAcceptConnections: true,
   logger: testLogger,
   indy,
-};
+}
 
 describe('with mediator', () => {
-  let aliceAgent: Agent;
-  let bobAgent: Agent;
-  let aliceAtAliceBobId: string;
+  let aliceAgent: Agent
+  let bobAgent: Agent
+  let aliceAtAliceBobId: string
 
   afterAll(async () => {
-    (aliceAgent.inboundTransporter as PollingInboundTransporter).stop = true;
-    (bobAgent.inboundTransporter as PollingInboundTransporter).stop = true;
+    ;(aliceAgent.inboundTransporter as PollingInboundTransporter).stop = true
+    ;(bobAgent.inboundTransporter as PollingInboundTransporter).stop = true
```

Comment:
I don't understand why semicolons are at the start of the line here? 🤔

Reply:
That's because we use `()`. Bit weird...